public/src/components/navbar/NavBarLabs.tsx

```diff
@@ -20,7 +21,7 @@ const NavBarLabs = (): JSX.Element | null => {
         return (
             <div>
                 {assignment.isGroupLab && <i className={"fa fa-users"} title={"Group assignment"} />}
-                {submission?.status === Submission_Status.APPROVED && <i className="fa fa-check ml-2" />}
+                {getStatusByUser(submission, state.self.ID) === Submission_Status.APPROVED && <i className="fa fa-check ml-2" />}
```

Comment:
Do we have a helper that checks if a submission is approved that we could use here instead of doing `===`?